requirements/test.txt

```diff
@@ -1,4 +1,5 @@
 -r base.txt
+aiofiles
```

Comment:
Why do you think this is needed in the tests?

Reply:
Reverted.  This will cause flake8 to raise an I900 error.
* https://github.com/Arkq/flake8-requirements#readme